modules/renter/renter.go

```diff
@@ -161,6 +161,9 @@ type Renter struct {
 	mu             *sync.RWMutex
 	tg             *sync.ThreadGroup
 	tpool          modules.TransactionPool
+
+	// Testing utilities
+	debugFlags map[string]bool
```

Comment:
As mentioned in the WAL, this pattern should be replaced with an explicit set of fields/bools that have names and comments explaining what the flags do. That makes it a lot easier to get oriented when you are first reading the codebase.